src/nodes/delta/layouts.jl

```diff
@@ -0,0 +1,143 @@
+# DeltaFn node non-standard rule layout for `DeltaUnscented` and `DeltaExtended` approximations rule
+# `DeltaUnscented` and `DeltaExtended` nodes change the default layout in 2 ways:
+# - `m_out` requires only inbounds messages on `m_in`s (TODO consider exchanging this behaviour with CVI in the future)
+# - (ONLY WITH KNOWN INVERSE) `m_in(k)` requires only inbound message on `m_out` and inbound messages on the remaining `m_in`s
+
+struct DeltaUEUknownInverseApproximationDeltaFnRuleLayout end
```

Comment:
what does the `UEU` part mean here?

Reply:
UE - Unscented/Extended :) The next U belongs to Unknown